main.py

```diff
@@ -59,16 +56,10 @@
     #     EXPERIMENT_ID='Experiment2_INFERENCE_ONLY',
     #     ONLY_INFERENCE=True,
     #     EPOCHS=5,
-    #     ROLE='Both',
+    #     ROLE='Car1',
     #     EXPLORATION_EXPLOTATION_THRESHOLD=0,
-    #     TIME_BETWEEN_STEPS=0.05,
-    #     LEARNING_RATE=1,
-    #     N_STEPS=160,
-    #     BATCH_SIZE=160,
-    #     FIXED_THROTTLE=0.75,
-    #     THROTTLE_FAST=1,
-    #     THROTTLE_SLOW=0.5,
-    #     LOAD_WEIGHT_DIRECTORY= get_latest_model("experiments")
+    #     LOAD_WEIGHT_DIRECTORY= get_latest_model("experiments"),
```

Comment:
Change get_latest_model to a function that specifies the model files you want to load, and not the latest.

Reply:
there is another function for that : get_model_from_specific_directory. i just prefer to use the function in this code.